README.md

```diff
@@ -308,6 +313,238 @@ getPerson.addResolver(api, {
 });
 ```
 
+### Event Bridge
+
+Functionless makes using Event Bridge easy by leveraging typescript instead of AWS Event Bridge's proprietary logic and transform configuration.
+
+Event Bridge can:
+
+- Create Rules (`EventBusRule`) on a Event Bus to match incoming events.
+- Transform the event before sending to some services like `Lambda` Functions.
+- Target other AWS services like Lambda and other Event Buses
+
+Functionless uses a wrapped version of CDK's Event Bus, lets create a CDK event bus first.
+
+```ts
+// Create a new Event Bus using CDK.
+const awsBus = new aws_events.EventBus(this, "myBus");
+
+// Functionless also supports using the default bus or importing an Event Bus.
+const defaultBus = EventBus.fromEventBusName(this, "defaultBus", "default");
+const importedBus = EventBus.fromEventBusArn(this, "defaultBus", arn);
+```
+
+Functionless supports well typed events, lets add our event schema to Typescript.
+
+```ts
+interface UserDetails {
+  id?: string;
+  name: string;
+  age: number;
+  interests: string[];
+}
+
+interface UserEvent
+  extends functionless.EventBusRuleInput<
+    UserDetails,
+    // We can provide custom detail-types to match on
+    "Create" | "Update" | "Delete"
+  > {}
+```
+
+And wrap our CDK `EventBus`
```

Comment:
Explain why? Also, I think we should add an ability to instantiate our constructs without wrapping - we can use constructor overloads to allow both?

Reply:
Let me try

Reply:
We can do the constructor change in a separate pr

Reply:
Already did it! But had to solve a problem where the functionless static members don't propagate to declarations automatically. 

Will push tonight